src/test/java/edu/cornell/kfs/concur/batch/fixture/ConcurSAEDetailLineFixture.java

```diff
@@ -189,13 +199,16 @@ private ConcurSAEDetailLineFixture(ConcurSAEDetailLineFixture baseFixture, Concu
                 overrideMap.getOrDefault(LineField.ORG_REF_ID, baseFixture.orgRefId),
                 journalDebitCredit,
                 journalAmount,
-                overrideMap.getOrDefault(LineField.REPORT_END_DATE, baseFixture.reportEndDate));
+                overrideMap.getOrDefault(LineField.REPORT_END_DATE, baseFixture.reportEndDate), 
+                overrideMap.getOrDefault(LineField.CASH_ADVANCE_CA_KEY, ""),
+                overrideMap.getOrDefault(LineField.REPORT_ENTRY_ID, ""));

```

Comment:
These should default to the baseFixture field values instead of defaulting to "".